incident/tests/test_filtering.py

```diff
@@ -2190,34 +2015,6 @@ def test_targeted_institution_filtering_via_targeted_journalists(self):
         self.assertEqual(set(incidents), {tj1.incident, with_institution1_target})
 
 
-class LegalOrderTypeFilterTest(TestCase):
-    desired_type = choices.LegalOrderType.MOTION_TO_COMPEL
-    other_type = choices.LegalOrderType.WARRANT
-
-    @classmethod
-    def setUpTestData(cls):
-        CategoryPageFactory(
-            title='Subpoena / Legal Order',
-            incident_filters=['legal_order_type'],
-        )
-        cls.legal_order1 = factories.LegalOrderFactory(
-            order_type=cls.desired_type,
-        )
-        factories.LegalOrderFactory(
-            order_type=cls.other_type,
-        )
-
-    def test_filters_by_legal_order_type(self):
-        incidents = IncidentFilter({
-            'legal_order_type': self.desired_type,
-        }).get_queryset(strict=True)
-
-        self.assertQuerysetEqual(
-            incidents,
-            [self.legal_order1.incident_page],
-        )
-
-
```

Comment:
Similarly to the `status_of_charges` filter, I think we will want to keep the `legal_order_type` filter, which uses the `LegalOrderTypeFilter` class, and that does not involve the `legal_order_type` field.